### ISSUE TITLE
Scrape data on education from 2017 report

### ISSUE THREAD

**Issue by ravinepal** [Code4Nepal/data#50]
Ministry of Education published a report in 2018: https://moe.gov.np/assets/uploads/files/Education_in_Figures_2017.pdf

Please scrape table 2 to 91 and submit a pull request by creating a folder "Education in Figures 2017" within "[Federal Data](https://github.com/Code4Nepal/data/tree/master/Federal%20Data)" folder. 

Please extract data in a CSV. 

@divyamani1 would you be interested in this project? 

We will use some of these data for our upcoming [federal version of NepalMap](https://github.com/Code4Nepal/nepalmap_federal)

**Comment by divyamani1** [comment]
Sure. I'll look into it soon. So, I have to create CSV files for each table from Table 2 to 91 in a "Education in Figures 2017" folder inside the Federal Data folder? And, is there any particular file naming conventions that I should follow?

**Comment by ravinepal** [comment]
Thank you for asking @divyamani1. Yes for a CSV file for each table. I'd recommend using the name of the Table for the name of respective CSV file. For example: Table 3 is "Social Groups by Province (Recorded in National Population and Housing Census, 2011)" so I'll suggest using the same name for the respective CSV file. Sounds good?

Really appreciate your interest in helping Nepalis have more access to data.

**Comment by divyamani1** [comment]
Okay. Is it alright if I upload these CSVs on Open Data Nepal? I have also been contributing to Open data Nepal as a Data wrangler.

**Comment by ravinepal** [comment]
Of course, it will be great to have these data upload to this repo and to Open data Nepal. Thank you in advance. 

letting @nikeshbalami know.

**Comment by nikeshbalami** [comment]
Hi Guys, This PDF has been already extracted by @Ashwin-Dhakal and all the datasets are available at Open Data Nepal, under Education category: http://opendatanepal.com/dataset?oknp_category_label=Education

We need to analyze the data and filter it according to the Local Level or Districts Level or Province Level so that it can be used on NepalMap. Please take Population and Household data as reference: https://github.com/Code4Nepal/data/tree/master/Federal%20Data/753%20Local%20Unit%20Population%20and%20HouseHold

**Comment by ravinepal** [comment]
Thanks @nikeshbalami and @Ashwin-Dhakal - that's great. 

@divyamani1 - if you would like you can look at [Open Data Nepal](http://opendatanepal.com/dataset?oknp_category_label=Education) and take provincial data of various indicators and submit pull request under "[Federal Data](https://github.com/Code4Nepal/data/tree/master/Federal%20Data)" folder as we discussed earlier. As Nikesh said lease take Population and Household data as reference so data you submit can be used by NepalMap easily: https://github.com/Code4Nepal/data/tree/master/Federal%20Data/753%20Local%20Unit%20Population%20and%20HouseHold

**Comment by nikeshbalami** [comment]
And please make sure its 77 districts.

**Comment by divyamani1** [comment]
Sure. I'll check it out.

**Comment by code4nepal** [comment]
@pratimakandel - could you pls look at education data here: http://opendatanepal.com/dataset?oknp_category_label=Education

and create a pull request in a format so we can display education data on province level on beta.nepalmap.org? The process should be similar to population data that you scraped and populated on beta.nepalmap.org 

Pls create a folder named "2011 Education Data" within [Federal Data](https://github.com/Code4Nepal/data/tree/master/Federal%20Data) and please pull requests for these datasets: 

1. - http://opendatanepal.com/dataset/ph-d-level-student-enrollment-by-faculty-province-ministry-of-education
2. - http://opendatanepal.com/dataset/m-phil-level-student-enrollment-by-faculty-province-ministry-of-education
3. - http://opendatanepal.com/dataset/masters-level-student-enrollment-by-faculty-province-ministry-of-education
4. - http://opendatanepal.com/dataset/bachelor-level-student-enrollment-by-faculty-province-ministry-of-education
5. http://opendatanepal.com/dataset/province-wise-student-enrollment-of-higher-education-by-level-and-sex-ministry-of-education
6. http://opendatanepal.com/dataset/sex-wise-student-enrollment-of-higher-education-by-province-ministry-of-education
7. http://opendatanepal.com/dataset/total-number-of-faculties-of-higher-education-by-province-ministry-of-education

After than submit pull requests to be these datasets to be populated on NepalMap: https://github.com/Code4Nepal/nepalmap_federal

Pls let me know if you have questions.
